src/sentry/models/group.py

```diff
@@ -422,6 +422,10 @@ def title(self):
         et = eventtypes.get(self.get_event_type())(self.data)
         return et.to_string(self.get_event_metadata())
 
+    def location(self):
```

Comment:
Is location required for groups? I can't see where it's used

Reply:
It's used in the UI but I figured it makes sense to provide that attribute for internal/plugin usage as well.